### DISCUSSION TITLE
The pyfrozen class example dosent work

### DISCUSSION THREAD

**Discussion by psvri** [PyO3/pyo3#3172] [Questions] [answered]
Hello Folks,

While going through the user guide for the [pyclass frozen](https://pyo3.rs/main/class.html#frozen-classes-opting-out-of-interior-mutability) I see that the code given in the example dosent compile and I get the error `no method namedgetfound for structpyo3::Py in the current scope` . Also a side note I dont find that function in the [docs as well](https://docs.rs/pyo3/latest/pyo3/pycell/index.html?search=fn%3Aget) . Can you let me know the fixes that needs to be done in the example for it to work ? .Below is the full code given in the docs

```
#![allow(unused)]
fn main() {
use std::sync::atomic::{AtomicUsize, Ordering};
use pyo3::prelude::*;

#[pyclass(frozen)]
struct FrozenCounter {
    value: AtomicUsize,
}

let py_counter: Py<FrozenCounter>  = Python::with_gil(|py| {
    let counter = FrozenCounter { value: AtomicUsize::new(0) };

    Py::new(py, counter).unwrap()
});

py_counter.get().value.fetch_add(1, Ordering::Relaxed);
}
```

**Comment by adamreichold** [comment, marked as answer]
I think you are looking at the guide for the `main` branch as indicated by your link but build against a released version of PyO3, probably 0.18.3? `Py::get` was merged only recently and is not yet part of a released version.

You can either read the guide for the released version at e.g. https://pyo3.rs/v0.18.3/class.html or build against the main branch using a Git dependency like

```toml
pyo3 = { git = "https://github.com/PyO3/pyo3.git" }
```

**Reply by adamreichold** [reply]
(But also note that the trait bounds still have a bug, i.e. https://github.com/PyO3/pyo3/pull/3169)

**Reply by psvri** [reply]
Oops, I failed to realize this.

Thanks for the help.
